### DISCUSSION TITLE
mpv 0.39 on Debian bookworm

### DISCUSSION THREAD

**Discussion by madhums** [mpv-player/mpv#15474] [Q&A]
Hi, has anyone here any experience installing mpv player latest version 0.39 on Debian using apt or deb files? I’ve tried compiling it with source but a lot of trouble getting it to work. Thanks for any tips or suggestions in advance. Also don’t understand why 0.39 isn’t available on apt yet…?

**Comment by kasper93** [comment]
> Hi, has anyone here any experience installing mpv player latest version 0.39 on Debian using apt or deb files?

https://github.com/mpv-player/mpv-build?tab=readme-ov-file#instructions-for-debian--ubuntu-package

> Also don’t understand why 0.39 isn’t available on apt yet…?

That would be a question for Debian package maintainer.

**Reply by kasper93** [reply]
> That would be a question for Debian package maintainer.

Debian sid seems to update to 0.39.0 yesterday.

**Comment by CounterPillow** [comment]
I'm a month late but I felt like I needed to respond to this:

> Also don’t understand why 0.39 isn’t available on apt yet…?

You seem to fundamentally misunderstand Debian. Debian ships outdated software on purpose. They will not ship an upstream feature update for an existing Debian stable release like Bookworm. mpv's developers do not control what Debian ships, this is left up to Debian. The "stable" here does not refer to the number of bugs, it just means they won't update things until the next Debian release unless it's a security fix that someone let Debian's package maintainer know about.

If you want up-to-date software, you'll want a rolling release distribution.
